### DISCUSSION TITLE
Consider integrating `termynal`

### DISCUSSION THREAD

**Discussion by adam-moss** [ministryofjustice/mkdocs-tech-docs-template#46] [Ideas]
tech-docs quite often have code snippets embedded.  It can be useful to render this as an animated terminal.

[termynal](https://github.com/ines/termynal) is a nice lightweight component that can provide this whilst failing gracefully for anyone with javascript disabled.

An example of it embedded within material-for-mkdocs is viewable throughout the [typer-cli](https://typer.tiangolo.com/#run-it) docs.

**Comment by SoumayaMauthoorMOJ** [comment]
Hey @adam-moss  thanks for raising this.  Could you clarify the ask?

**Comment by adam-moss** [comment]
To provide an interactive representation of the REPL on the terminal, repo rather than just static content, if js is enabled.

**Comment by adam-moss** [comment]
Actually there is a relatively new mkdocs plugin that can do this https://github.com/daxartio/termynal

**Comment by SoumayaMauthoorMOJ** [comment]
Cool in that case I'll close the ticket. Feel free to raise an idea https://github.com/ministryofjustice/mkdocs-tech-docs-template/discussions/categories/ideas as a suggestion for mkdocs plugins that can be helpful :-)

**Comment by SoumayaMauthoorMOJ** [comment]
I've  reponed the issue and converted to discussion :-)
